swap_meet/vendor.py

```diff
@@ -1,2 +1,68 @@
+from swap_meet.item import Item
 class Vendor:
-    pass
\ No newline at end of file
+    
+    def __init__(self, inventory=None):
+        # self.inventory = inventory
+        if inventory is None:
+            inventory = []
+        self.inventory = inventory
+    
+    def add(self, item):
+        self.item = item
+        self.inventory.append(item)
+        return item
+    def remove(self, item):
+        self.item = item
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+        return False
```

Comment:
Prefer writing the error handling case as a guard clause first, which lets us then emphasize the main part of the code by not indenting it.
```py
        if item not in self.inventory:
            return False

        self.inventory.remove(item)
        return item
```

We could also do this with a more EAFP approach by using `remove` or `index` directly and handling the error that would get raised if the item were not present (`ValueError`).